interactive_engine/executor/ir/core/src/glogue/pattern.rs

```diff
@@ -456,24 +485,32 @@ impl Pattern {
                 }
             }
         }
+        if edge_expands_to_close.len() > 0 {
+            Err(IrPatternError::Unsupported(
+                "Some Expand Edge Only Operator doesn't followed by GetV to close".to_string(),
+            ))?
+        }
         let mut pattern = if !pattern_edges.is_empty() {
             Pattern::from(pattern_edges)
         } else {
-            if vertex_data_map.is_empty() {
+            if vertex_params_map.is_empty() {
```

Comment:
btw, what if all vertex_params are None, then vertex_params_map is empty?

Reply:
This situation only happens there are only `as()` step in the match, then it is actually can be handled by the naive strategy